scripts/fix.sh

```diff
@@ -0,0 +1,13 @@
+GO_CMD="go"
+
+# TODO(ptabor): Expand to cover different architectures (GOOS GOARCH), or just list go files.
```

Comment:
Is this TODO needed?

Reply:
I didn't modify the file when I took it from the etcd repo. :( But thanks for pointing this out, I will remove it to keep it clean! :) 

Thank you! :)